src/main/java/graphql/execution/Execution.java

```diff
@@ -96,6 +96,7 @@ public Execution(ExecutionStrategy queryStrategy, ExecutionStrategy mutationStra
                 .cacheControl(executionInput.getCacheControl())
                 .locale(executionInput.getLocale())
                 .valueUnboxer(valueUnboxer)
+                .executionInput(executionInput)
```

Comment:
We need this later in subscriptions and the `instrumentDataFetcherResult` calls.   Its nominally useful as well but execution context unpacks it any way